resources/assets/enderio/lang/de_DE.lang

```diff
@@ -290,11 +587,18 @@ tile.blockTravelAnchor.tooltip.detailed.line1=Erlaubt Teleport zu anderen Ankern
 tile.blockTravelAnchor.tooltip.detailed.line2=Stehe auf einem Anker, schaue in Richtung
 tile.blockTravelAnchor.tooltip.detailed.line3=eines anderen und springe zum Reisen
 tile.blockTravelAnchor.tooltip.detailed.line4=Kann auch mit dem Stab des Reisens
-tile.blockTravelAnchor.tooltip.detailed.line5=verwendet werden
+tile.blockTravelAnchor.tooltip.detailed.line5=verwendet werden.
+tile.blockTravelAnchor.tooltip.detailed.line6=Kann in einer Färbemaschine gefärbt werden.
 
 enderio.blockTravelPlatform.invalidTarget=Ziel ist blockiert
 enderio.blockTravelPlatform.outOfRange=Ziel ist außer Reichweite
 
+tile.blockTelePad.name=Telepad-Block
+tile.blockTelePad.tooltip.detailed.line1=Benutzt Energie um Entitäten zu einer
```

Comment:
Okay, here is a cheat I found in my Carpenter's Blocks translation: "Benutzt Energie, alles mögliche zu anderen Orten und Dimensionen zu teleportieren."

What do you think?